integration_tests/commands/async/set_test.go

```diff
@@ -100,21 +107,11 @@ func TestSetWithOptions(t *testing.T) {
 			commands: []string{"SET k2 v2 PXAT 123123", "GET k2"},
 			expected: []interface{}{"OK", "(nil)"},
 		},
-		{
-			name:     "XX on existing key",
-			commands: []string{"SET k v1", "SET k v2 XX", "GET k"},
-			expected: []interface{}{"OK", "OK", "v2"},
-		},
 		{
 			name:     "Multiple XX operations",
 			commands: []string{"SET k v1", "SET k v2 XX", "SET k v3 XX", "GET k"},
 			expected: []interface{}{"OK", "OK", "OK", "v3"},
 		},
-		{
-			name:     "EX option",
-			commands: []string{"SET k v EX 1", "GET k", "SLEEP 2", "GET k"},
-			expected: []interface{}{"OK", "v", "OK", "(nil)"},
-		},
```

Comment:
Why have we removed these?

Reply:
The former one was moved up for better readability and the later one is a duplicate.